tests/trainer/properties/log_dir.py

```diff
@@ -140,3 +140,24 @@ def test_logdir_custom_logger(tmpdir):
     assert trainer.log_dir == expected
     trainer.fit(model)
     assert trainer.log_dir == expected
+
+
+def test_logdir_logger_collection(tmpdir):
+    """
+    Tests that the logdir equals the default_root_dir when the logger is a
+    LoggerCollection
+    """
+    expected = tmpdir
+    model = TestModel(expected)
+
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        max_steps=2,
+        callbacks=[ModelCheckpoint(dirpath=tmpdir)],
```

Comment:
Is a `ModelCheckpoint` callback necessary for the test?

Reply:
Nope---I've removed it.